### DISCUSSION TITLE
Installing  FlareSolverr on Qnap

### DISCUSSION THREAD

**Discussion by Scar13t** [FlareSolverr/FlareSolverr#645] [Q&A]
Was wondering how I'd go about installing FlareSolverr on my qnap TVS-471

Would I need to install a docker container from container station?

**Comment by ilike2burnthing** [comment]
Yes.

**Reply by Scar13t** [reply]
How do I get docker on my qnap? I saw I could use container station but the peeps over at prowlarr were talking about how I shouldn't use that no matter what

**Reply by ilike2burnthing** [reply]
As far as I can tell, Container Station is your only option, and it looks fine.

**Reply by Scar13t** [reply]
Okay which docker container should I get? There are different variations of it on container station

**Reply by ilike2burnthing** [reply]
Latest stable is v2.2.10. Latest beta is v3.

**Reply by Scar13t** [reply]
Those are not my options
![image](https://user-images.githubusercontent.com/80722285/210419291-8589aee2-7b01-49c7-af06-e915bfd65f0b.png)
![image2](https://user-images.githubusercontent.com/80722285/210419302-02db8027-e8af-4318-be74-ec2e01495351.png)

**Reply by ilike2burnthing** [reply]
You already have Docker installed, that's what Container Station is the frontend for. You want to install FlareSolverr.

**Comment by ngosang** [comment]
Official Docker image
https://hub.docker.com/r/flaresolverr/flaresolverr
